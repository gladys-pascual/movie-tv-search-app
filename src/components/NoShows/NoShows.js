import React from "react";

const NoShows = () => {
  return (
    <section className="no-shows">
      <span className="material-icons">sentiment_very_dissatisfied</span>
      <h3>
        Sorry, no movie or TV show matched your search. Please type your search
        again.
      </h3>
    </section>
  );
};

export default NoShows;
