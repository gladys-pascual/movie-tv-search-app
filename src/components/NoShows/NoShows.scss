@import "../../assets/variables.scss";

.no-shows {
  color: $theme-color;
  height: 140px;
  padding-top: 62px;
  text-align: center;
  font-size: 1.25em;

  .material-icons {
    padding-right: 10px;
    font-size: 6em;
  }
}
