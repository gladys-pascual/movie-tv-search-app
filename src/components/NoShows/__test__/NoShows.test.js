import React from "react";
import { render } from "@testing-library/react";
import NoShows from "../NoShows";

test("renders NoShows component as expected", () => {
  const { container } = render(<NoShows />);
  expect(container).toMatchSnapshot();
});
