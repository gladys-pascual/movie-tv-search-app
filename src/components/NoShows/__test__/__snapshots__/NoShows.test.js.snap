// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NoShows component as expected 1`] = `
<div>
  <section
    class="no-shows"
  >
    <span
      class="material-icons"
    >
      sentiment_very_dissatisfied
    </span>
    <h3>
      Sorry, no movie or TV show matched your search. Please type your search again.
    </h3>
  </section>
</div>
`;
